### ISSUE TITLE
Weird DeoVR issue

### ISSUE THREAD

**Issue by jwvanderbeck** [xbapps/xbvr#429]
First, thanks for adding support for DeoVR this is awesome!

However I am having an odd issue with videos from one specific site.  And actually only certain videos.  The odd thing is these videos play fine in DeoVR on its own but do NOT play when using the XBVR integration.  DeoVR just throws an error.  Also the higher quality version of these files do play through DeoVR via XBVR.

It is kind of weird.  I'm not sure what I can do to provide more info as obviously I can't share the videos as they are from a pay site.

tl;dr the videos play properly through DeoVR stand-alone, but not through DeoVR via XBVR.

**Comment by cld9x** [comment]
@jwvanderbeck fixed in 0.3.7

**Comment by jwvanderbeck** [comment]
This was **not** the `categories` issue.  I had seen an issue opened for that and wouldn't have duplicated it.

This was something else and continues to still not work on 0.3.7 just tested.
